index.html

```diff
@@ -37,6 +37,21 @@ <h1>advantages of being a webdeveloper in cameroon</h1>
         <li>your are regarded as a genuise lol</li>
         <li>your have a limitless oppotunity</li>
     </ul>
-    
+    <div>
+        <h2>How much do u plan making a year in <span>webdevelopment field</span></h2>
+        <p>umm between 80k$ to 200k$</p>
+        <p>
+            the 80k$ is when i am starting out,
+            when they will recognice my potencial they will 
+            upp my salery... <br>
+            but i dont plan making all that money from a company alone 
+            in a year... <br>
+            i head as a webdeveloper in usa u make 100k to 130k a year.. <br>
+            i plan making that money working for a company and my site hustle <br>
+            <strong>i will be a millionair in dollers in 2025</strong>
```

Comment:
i strongly believe this and i am working toward it, if you can help me get there earlier it will be a big win